resources/GraphQL.js

```diff
@@ -1,6 +1,24 @@
-const { readFileSync: read } = require('fs');
+const { readFileSync: read, readdirSync: readdir } = require('fs');
+const path = require('path');
+
+function loadGraphQL(target, type) {
+  let dir = `${__dirname}/graphql/${type}`
+  readdir(dir, {withFileTypes: true}).forEach((e) => {
+    if (e.isFile() && path.extname(e.name) == '.graphql') {
+      let name = path.basename(e.name, '.graphql');
+      target[name] = read(`${dir}/${e.name}`).toString();
+    }
+  });
+}
+
+let queries = {};
+let mutations = {};
+loadGraphQL(queries, "queries");
+loadGraphQL(mutations, "mutations");
 
 module.exports = Object.freeze({
-  get EVALUATE_CODE_QUERY() { return read(`${__dirname}/graphql/queries/evaluateCodeQuery.graphql`).toString(); },
-  get REDEEM_CODE_MUTATION() { return read(`${__dirname}/graphql/mutations/redeemCodeMutation.graphql`).toString(); },
+  queries: Object.freeze(queries),
+  mutations: Object.freeze(mutations),
+  get EVALUATE_CODE_QUERY() { return queries.evaluateCodeQuery; },
```

Comment:
I assume this is kept to support older code?

Reply:
Yeah, I didn't want to break existing code if anyone relies on using these.
But if we bump major version we can remove them.

Reply:
Yeah, I agree.